Patch to ignore -little to allow building on SH4 architecture.

Index: elmerfem/fem/acx_elmer.m4
===================================================================
--- elmerfem.orig/fem/acx_elmer.m4	2010-05-04 18:37:43.000000000 -0400
+++ elmerfem/fem/acx_elmer.m4	2010-05-04 18:38:25.000000000 -0400
@@ -772,7 +772,7 @@
 	   acx_cxx_libs="$acx_cxx_libs $acx_cxx_ldflag"
 	;;
         # Ignore these flags.
-        -lang* | -lcrt*.o | -lc | -lgcc | -libmil | -LANG:=*)
+        -lang* | -lcrt*.o | -lc | -lgcc | -libmil | -little | -LANG:=*)
 	;;
 	-[[lLR]]*)
 dnl	   echo "$acx_cxx_ldflag"
